starters/cms/src/data.ts

```diff
@@ -0,0 +1,174 @@
+import type { CollectionItemData, ManagedCollection, ManagedCollectionField } from "framer-plugin"
+
+import { framer } from "framer-plugin"
+import { PLUGIN_KEYS } from "./constants"
+
+export type ManagedCollectionFieldType = ManagedCollectionField["type"]
+export type ManagedCollectionFields = readonly ManagedCollectionField[]
+export type FieldIds = ReadonlySet<ManagedCollectionField["id"]>
```

Comment:
I am not a big fan of creating these types. It adds another layer of abstraction away from the core APIs and should be up to the author to decide if they want to do this.

We should only do this if its saving a lot of lines of code